dpctl/_sycl_context.pyx

```diff
@@ -165,3 +168,17 @@ cdef class SyclContext(_SyclContext):
             SyclContext cast to a size_t.
         """
         return int(<size_t>self._ctx_ref)
+
+    def get_devices (self):
+        cdef DPCTLDeviceVectorRef DVRef = DPCTLContext_GetDevices(self.get_context_ref())
```

Comment:
DVRef needs to be freed.

Reply:
Changed in 4a3e4b3eae658a79eab77ae6601e3fe688959ade